### PR TITLE
community: Add bio for Liza Babu

### PR DESCRIPTION
**Pull request by lizababu**

### CODE REVIEW

File: website/docs/community/people.md [hunk 1]
```diff
@@ -4,4 +4,16 @@ title: People
...
-This is content
+## Liza Babu
+
+![Liza Babu](../../static/img/people/liza_babu.jpeg)
+
+Liza is a contributor to the OpenEdu project (21-COP-0016).
+She contributes beyond the scenes with ideas to the methodology of the project and takes care of the project overall presentation.
+
+Liza finished her Masters degree in Advanced Cybersecurity in 2019 at University POLITEHNICA of Bucharest, Romania.
+She is also a teaching assistant at the University POLITEHNICA of Bucharest, being involved in Computer Programming, Operating Systems and Hardware-Software Interface classes.
```
Comment (razvand) [inline website/docs/community/people.md]: I'd say without `the`.

File: website/docs/community/people.md [hunk 2]
```diff
@@ -4,4 +4,16 @@ title: People
...
+She is interested in operating systems, security, low-level programming and a big fan of open source.
+She loves spending time with students, teaching and finding ways to improve the current education system.
+
+Liza also currently works at [Bitdefender](https://www.bitdefender.com/) as a software engineer, part of the [Connected Home Security](https://www.bitdefender.com/smart-home/) team.
```
Comment (razvand) [inline website/docs/community/people.md]:
```suggestion
Liza currently works at [Bitdefender](https://www.bitdefender.com/) as a software engineer, part of the [Connected Home Security](https://www.bitdefender.com/smart-home/) team.
```

**Review by razvand — COMMENTED** [review]
Some minor comments.

**Comment by razvand** [comment]
I would use "I" instead of "She", as we discussed last time.

File: website/docs/community/people.md (hunk 1 above)
Comment (lizababu) [reply website/docs/community/people.md]: I agree, I left it here to see your opinion (I saw Teo's comment on your PR).

File: website/docs/community/people.md [hunk 3]
```diff
@@ -4,4 +4,17 @@ title: People
...
 # Project Team
 
-This is content
+## Liza Babu
+
+![Liza Babu](../../static/img/people/liza_babu.jpeg)
+
+Hello!
+I'm Liza, I am a contributor to the OpenEdu project (21-COP-0016).
+I operate beyond the scenes with ideas to the methodology of the project and takes care of the project overall presentation.
```
Comment (RazvanN7) [inline website/docs/community/people.md]:
```suggestion
I operate beyond the scenes with ideas to the methodology of the project and take care of the project overall presentation.
```

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**